scripts/menu.ps1

```diff
@@ -0,0 +1,22 @@
+param (
+    [string]$server = "https://localhost:5001"
+)
+$item = Read-Host @"
+1: Get pet by id.
+2: Unavailable
+3: Unavailable
+Please select the task [1-3] 
+"@
+switch ($item)

```

Comment:
switch should also have default behavior

Reply:
Done.